pkgs/development/python-modules/duckduckgo-search/default.nix

```diff
@@ -13,24 +13,24 @@
 
 buildPythonPackage rec {
   pname = "duckduckgo-search";
-  version = "6.3.2";
+  version = "6.3.4";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "deedy5";
     repo = "duckduckgo_search";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-5AuPAv78ePrnCr5L4CfIu/fq7Ha19zC78zg8JLu3U2A=";
+    rev = "v${version}";
```

Comment:
```suggestion
    rev = "refs/tags/v${version}";
```

Reply:
Why tho? The output is the same right? Why does this matter? A Git hash cannot be confused with a version right?

Reply:
The output is the same indeed and the git commit can be confused with a version tag. This is actually the best practices in `nixpkgs`.

Also, when submitting a PR, it's a good practice to try always submit the least minimal changes that fulfil your initial PR. In this case, it is just a simple bump, so nothing else has to be changed.

We could be much more picky (like the `click` line change), but I'm not that kind of person :)

Reply:
Ooh ok I did not know :) I will change it as soon as possible